.github/workflows/release.yml

```diff
@@ -5,7 +5,7 @@ name: Check and realese cosmpy
 # secrets PYPI_USERNAME and PYPI_PASSWORD are required!
 on:
   push:
-    branches: [ master ]
+    branches: [ 'release/**' ]
```

Comment:
This seems wrong. This would trigger on PR from branches that go into a branch named `release/...`. 

We need the source branch name to be `release/...` and not the target branch name. 
Please see https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-pull_request-workflow-based-on-the-head-or-base-branch-of-a-pull-request